src/components/FileManager/FileName-utils.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * The contents of this file are subject to the Common Public Attribution License Version 1.0 (the “License”); you may not use this file except in compliance with the License. You may obtain a copy of the License at https://mpapp-public.gitlab.io/manuscripts-frontend/LICENSE. The License is based on the Mozilla Public License Version 1.1 but Sections 14 and 15 have been added to cover use of software over a computer network and provide for limited attribution for the Original Developer. In addition, Exhibit A has been modified to be consistent with Exhibit B.
+ *
+ * Software distributed under the License is distributed on an “AS IS” basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the specific language governing rights and limitations under the License.
+ *
+ * The Original Code is manuscripts-frontend.
+ *
+ * The Original Developer is the Initial Developer. The Initial Developer of the Original Code is Atypon Systems LLC.
+ *
+ * All portions of the code written by Atypon Systems LLC are Copyright (c) 2024 Atypon Systems LLC. All Rights Reserved.
+ */
+export const trimFilename = (filename: string, maxLength: number) => {
```

Comment:
Do we need to have this in a new file?

Reply:
@mnatsheh 
I think it would be better to place this in a separate file as a utility, making it reusable in other parts of the product if needed. This way, we can easily access this functionality wherever it's required.

Reply:
I'm not really sure how useful this would be elsewhere, but if you prefer to have it separate, please move it to `lib/files.ts`.

Reply:
I was initially trying to follow to the standard like: footnotes. We have a footnote utility with the necessary functions, and I thought that truncating names might be used frequently in the editor. However, moving this functionality to `lib/files.ts` seems like a better approach. I'll make that change.

Reply:
done